tools/clang/tools/dxildll/CMakeLists.txt

```diff
@@ -58,6 +58,7 @@ target_link_libraries(dxildll PRIVATE
   dxcvalidator
 )
 
+if (HLSL_BUILD_DXILDLL)
```

Comment:
We could use HLSL_BUILD_DXILDLL guard add_subdirectory(dxildll) in tools/clang/tools/CMakeLists.txt.